pandas/tests/window/test_rolling.py

```diff
@@ -38,17 +38,26 @@ def test_constructor(self, which):
         c(window=2, min_periods=1, center=False)
 
         # GH 13383
-        with pytest.raises(ValueError):
+
+        msg = "window must be non-negative"
+
+        with pytest.raises(ValueError, match=msg):
```

Comment:
the c(0) below looks misplaced. It doesn't raise. can you co-locate with the #valid statements above while your touching this.